ts/components/conversation/composition/CompositionTextArea.tsx

```diff
@@ -84,7 +87,10 @@ function useMembersInThisChat(): Array<SessionSuggestionDataItem> {
   }
   const members = isPrivate
     ? uniq([UserUtils.getOurPubKeyStrFromCache(), selectedConvoKey])
-    : membersFor03Group || [];
+    : PubKey.is03Pubkey(selectedConvoKey)
+      ? membersFor03Group || []
+      : membersFor05LegacyGroup || [];
```

Comment:
`useLibGroupMembers` already returns `[]` if there are no members

Should we not do the same for `use05GroupMembers` and then you should not need the `|| []`?